### PR TITLE
add newrelic, refactor database

### PR DESCRIPTION
**Pull request by uxsasukexnaruto**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    entry: __dirname + '/client/src/index.jsx',
+    entry: `${__dirname}/client/src/index.jsx`,
```
Comment (wonil-park) [inline webpack.config.js:2]: Path.resolve(__dirname, "client", "src", "index.jsx")

File: webpack.config.js [hunk 2]
```diff
@@ -8,24 +8,26 @@ module.exports = {
...
-          },}],
+          },
+          }],
         }
       ]
     },
      output: {
       filename: 'bundle.js',
-      path: __dirname + '/client/dist'
+      path: `${__dirname}/client/dist`,
```
Comment (wonil-park) [inline webpack.config.js:31]: Recommend using Path.resolve or Path.join

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+const axios = require('axios');
 import $ from 'jquery';
```
Comment (wonil-park) [inline client/src/components/App.jsx:3]: recommend removing this line

File: client/src/components/App.jsx [hunk 4]
```diff
@@ -13,17 +14,24 @@ class App extends React.Component {
...
         this.displaySpaceInfo = this.displaySpaceInfo.bind(this);
     }
 
     componentDidMount() {
-        $.get({
-            url: `/listings/${window.location.href.match(/id\s*=\s*(.*)/)[1]}`,
-            success: (data) => {
-                this.setState({listings: data});
-            }
+        // $.get({
```
Comment (wonil-park) [inline client/src/components/App.jsx:23]: Recommend deleting comments

Comment (uxsasukexnaruto) [reply client/src/components/App.jsx:23]: got it, thanks

**Pull request merged**